joblib/_store_backends.py

```diff
@@ -348,6 +348,8 @@ def _get_items_to_delete(
 
         if age_limit is not None:
             older_item = min(item.last_access for item in items)
+            assert age_limit.total_seconds() >= 0, (
+                "age_limit has to be a positive timedelta")
```

Comment:
`AssertionError` are not specific enough, I would prefer that we raise a `ValueError`.

Reply:
Done in 053de30